pytest.ini

```diff
@@ -0,0 +1,11 @@
+[pytest]
+# Configure pytest to fail any test that raises an unexpected warning.
+# This will enable us to catch usages of deprecated features in downstream libraries,
+# before they are hard-deprecated.
+# Furthermore, they will catch any test issues which may hide bugs.
+# The format is `action:message_regex:category:module:line`.
+filterwarnings =
+    # Fail builds on any unexpected warnings.
+    # See https://docs.google.com/document/d/1TVMfmhO0vD1MdkVrqRS9t4om2shMTY9nqdqqOopNv0M for more details.
+    error
+    ignore:.*:
```

Comment:
this means, treat every warning, except for warnings that match `.*`, ignore those. this triggers the SyntaxError (which I fix in this PR) but keeps the same warning behavior until the next PR https://github.com/ray-project/ray/pull/31219